contrib/win32/win32compat/misc.c

```diff
@@ -1413,3 +1457,22 @@ get_others_file_permissions(wchar_t * file_name, int isReadOnlyFile)
 		LocalFree(pSD);
 	return ret;
 }
+
+char*
+get_ssh_dir_path()
+{
+	if (ssh_dir_path[0]) return ssh_dir_path;
+
+	int return_val = ExpandEnvironmentStringsA("%programData%", ssh_dir_path, PATH_MAX);
+	if (return_val > PATH_MAX)
+		fatal("%s, buffer too small to expand:%s", __func__, "%programData%");
+	else if (!return_val)
+		fatal("%s, failed to expand:%s error:%s", __func__, "%programData%", GetLastError());
+
+	// ssh_root_path - "c:\\ProgramData\\openssh\\"
+	char *p = "\\openssh";
+	strcat_s(ssh_dir_path, _countof(ssh_dir_path), p);
+	debug("__PROGRAMDATA__:%s", ssh_dir_path);
```

Comment:
Why log __PROGRAMDATA__: c:\ProgramData\openssh\?

Reply:
This will help in debugging for end users..